R/zchunk_LB133.ag_Costs_USA_C_2005.R

```diff
@@ -1,20 +1,26 @@
 #' module_aglu_LB133.ag_Costs_USA_C_2005
 #'
-#' Briefly describe what this chunk does.
+#' This module computes costs for each GCAM commodity (used by each region-GLU) using USDA cost
+#' information for USA when available. For commodities without USDA cost data, the average profit
+#' among USDA commodities and LDS harvested area and production information are used to compute
+#' cost.
 #'
 #' @param command API command to execute
 #' @param ... other optional parameters, depending on command
 #' @return Depends on \code{command}: either a vector of required inputs,
 #' a vector of output names, or (if \code{command} is "MAKE") all
 #' the generated outputs: \code{L133.ag_Cost_75USDkg_C}. The corresponding file in the
 #' original data system was \code{LB133.ag_Costs_USA_C_2005.R} (aglu level1).
-#' @details Describe in detail what this chunk does.
+#' @details USDA cost information is scaled to the level of GCAM commodity using LDS harvested area and
+#' production data to compute weighting factors for aggregation and convert cost from USD/square meter
+#' to the more useful USD/kg. Cost information is converted to the GCAM base of 1975 USD before aggregation.
+#' For commodities without USDA cost information, the average profit among USDA commodities is combined
+#' with LDS harvested area and production data to calculate cost.
 #' @importFrom assertthat assert_that
 #' @importFrom dplyr filter mutate select
 #' @importFrom tidyr gather spread
-#' @author YourInitials CurrentMonthName 2017
-#' @export
```

Comment:
That's so weird, it's done on my local branch. I'll try redoing the documentation, build and reload.